net/dnsfallback/dnsfallback.go

```diff
@@ -310,9 +310,9 @@ func bootstrapDNSMap(ctx context.Context, serverName string, serverIP netip.Addr
 // https://derp10.tailscale.com/bootstrap-dns
 type dnsMap map[string][]netip.Addr
 
-// getDERPMap returns some DERP map. The DERP servers also run a fallback
+// GetDERPMap returns some DERP map. The DERP servers also run a fallback
 // DNS server.
```

Comment:
Now that this is public, mind updating the comment? Something like "returns a DERP map that can be used for basic bootstrapping", and maybe mention that this shouldn't be used if there's a "real" one available in localbackend?

Reply:
Done.